viam-orb-slam3.go

```diff
@@ -41,7 +40,7 @@ var (
 	cameraValidationMaxTimeoutSec = 30 // reconfigurable for testing
 	dialMaxTimeoutSec             = 30 // reconfigurable for testing
 	// Model specifies the unique resource-triple across the rdk.
-	Model             = resource.NewModel("viam", "slam", "orbslamv3")
+	Model             = resource.NewModel("viam", "slam", "orbslam3")
```

Comment:
The scope says "orbslamv3", so this should stay as is.

Reply:
Oh -- Okay. I just thought it was a bit confusing how we are using orbslam3 everywhere else except for this one spot. I'll revert.